cmd/skaffold/app/cmd/cmd.go

```diff
@@ -133,10 +133,11 @@ func FlagToEnvVarName(f *pflag.Flag) string {
 
 func AddRunDeployFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&opts.Tail, "tail", false, "Stream logs from deployed objects")
+	cmd.Flags().BoolVar(&opts.ForceDeploy, "force", false, "Recreate resources if necessary")
 	cmd.Flags().StringArrayVarP(&opts.CustomLabels, "label", "l", nil, "Add custom labels to deployed objects. Set multiple times for multiple labels.")
 }
 
-func AddRunDevFlags(cmd *cobra.Command) {
+func AddGeneralFlags(cmd *cobra.Command) {
```

Comment:
I don't agree with this rename - what is your reasoning behind it? The idea in `AddRunDevFlags` was to name the skaffold commands - run and dev - in that the flags apply for. That is similar in `AddRunDeployFlags` above. `AddGeneralFlags` seems to promise a wider applicability - while not changing in functionality and being inconsistent with `AddRunDeployFlags`.

Reply:
Well, `AddRunDeployFlags` is exactly what it is says: it is only used in the run and the deploy commands.
On the other hand, `AddRunDevFlags` is used by build, delete, deploy, dev, and run. I find that the current name does not adequately convey the scope of this function. But you are right, that the new name is not perfect either. Maybe you have a better idea? Otherwise, I'll simply remove that commit.

Reply:
Alright, I don't want to be stubborn and dropped that commit  m(_ _)m